org.gnome.innstereo.json

```diff
@@ -0,0 +1,90 @@
+{
+    "app-id": "org.gnome.innstereo",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
+    "sdk": "org.gnome.Sdk",
+    "command": "innstereo",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.gfortran-62"
+    ],
+    "build-options": {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "env": {
+            "V": "1",
+            "PATH": "/app/bin:/usr/bin:/usr/lib/sdk/gfortran-62/bin"
+        }
+    },
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "gfortran",
+            "buildsystem": "simple",
+            "build-commands": [ "/usr/lib/sdk/gfortran-62/install.sh" ]
+        },
+        {
+            "name": "lapack",
+            "buildsystem": "cmake",
+            "builddir": true,
```

Comment:
Not needed.

Reply:
When I remove that line I get this error:

> lapack should not be configured & built in the lapack source directory
> You must run cmake in a build directory.